tmt/steps/provision/__init__.py

```diff
@@ -2609,21 +2662,34 @@ def _run_ansible(
 
         playbook = self._sanitize_ansible_playbook_path(playbook, playbook_root)
 
+        ansible_command = Command(
+            'ansible-playbook', *self._ansible_verbosity(), *self._ansible_extra_args(extra_args)
+        )
+
         # FIXME: cast() - https://github.com/teemtee/tmt/issues/1372
         parent = cast(Provision, self.parent)
 
+        provision_step = parent.plan.provision
+        inventory_path = provision_step.workdir / 'inventory.yaml'  # type: ignore[operator]
```

Comment:
1. `provision_step` does not need to exist, `inventory_path = parent.plan.provision.workdir / 'inventory.yaml'` would as nicely.
2. `ignore[operator]` is suspicious, why is it needed? It suggests there is something unexpected happening when creating the path.